### PR TITLE
Make compatible with Python 3

### PR DESCRIPTION
**Pull request by anderspitman**
Should just need a new qiime-default-reference release

### CODE REVIEW

**Comment by anderspitman** [comment]
Actually still need to run `runipynbs.py`...

**Comment by jairideout** [comment]
@anderspitman I'll work on a qiime-default-reference release

**Comment by jairideout** [comment]
@anderspitman release is live, so updating setup.py to require qiime-default-reference >= 0.1.3 should do the trick!

**Comment by anderspitman** [comment]
Ready for review.

File: fundamentals/msa-assignment.ipynb [hunk 1]
```diff
@@ -48,11 +48,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```
Comment (jairideout) [inline fundamentals/msa-assignment.ipynb]: Since this is an assignment you should clear all output.

Comment (anderspitman) [inline fundamentals/msa-assignment.ipynb]: Fixed

**Pull request merged**